segments/gitstatus/gitstatus.p9k

```diff
@@ -0,0 +1,149 @@
+# vim:ft=zsh ts=2 sw=2 sts=2 et fenc=utf-8
+################################################################
+# @title powerlevel9k Segment - Date
+# @source [powerlevel9k](https://github.com/bhilburn/powerlevel9k)
+##
+
+(){
+  # Set the right locale to protect special characters
+  local LC_ALL="" LC_CTYPE="en_US.UTF-8"
+
+  ################################################################
+  # Register segment helper icons
+  # Parameters:
+  #   name_of_icon  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
+  #                                          ●                                                                                  
+  p9k::register_icon "GITSTATUS_UNSTAGED"          $'\u25CF'   $'\uE17C'   $'\uF06A'   '\u'${CODEPOINT_OF_AWESOME_EXCLAMATION_CIRCLE}    $'\uF06A'
+#   #                                          ?                                                                                  
+#   p9k::register_icon "GITSTATUS_UNTRACKED"         '?'         $'\uE16C'   $'\uF059'   '\u'${CODEPOINT_OF_AWESOME_QUESTION_CIRCLE}       $'\uF059'
```

Comment:
Any reason why this is commented out?

Reply:
Yep. The reason is that there is a collision with `p9k::register_segment`. This function can optionally register an icon under a given state name. This name then cannot be used for `p9k::register_icon`. This is the case here: We already have a state `UNTRACKED`.. This bugs me and is on my to refactor list in my head.. I have tripped over that more than once.

Reply:
Oh, I see.